test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9197,11 +9211,12 @@ def evaluate_formula(self, jobid=None, formula=None, full=True,
                     f_value['eligible_time'] = self.utils.convert_duration(
                         job['eligible_time'])
             if 'fair_share_perc' in fres:
-                if self.scheduler is None:
-                    self.scheduler = Scheduler(server=self)
+                dflt_sched = self.schedulers[self.dflt_sched_name]
+                if dflt_sched is None:
+                    dflt_sched = Scheduler(server=self)
 
-                if 'fairshare_entity' in self.scheduler.sched_config:
-                    entity = self.scheduler.sched_config['fairshare_entity']
+                if 'fairshare_entity' in dflt_sched.sched_config:
```

Comment:
Will do these changes as separate ticket. 

Restrict the ticket to only create/delete/modify.